Source/SpatialGDK/Private/EngineClasses/SpatialNetBitWriter.cpp

```diff
@@ -18,14 +18,14 @@ void FSpatialNetBitWriter::SerializeObjectRef(UnrealObjectRef& ObjectRef)
 	*this << ObjectRef.Entity;
 	*this << ObjectRef.Offset;
 
-	uint8 HasPath = !ObjectRef.Path.empty();
+	uint8 HasPath = !ObjectRef.Path.IsSet();
```

Comment:
This is opposite of what it should be (`uint8 HasPath = ObjectRef.Path.IsSet();`)